pkgs/games/minecraft/default.nix

```diff
@@ -1,50 +1,95 @@
-{ stdenv, fetchurl, makeDesktopItem, makeWrapper
-, jdk, jre, libpulseaudio, libXxf86vm
+{ stdenv, fetchurl, dpkg, makeWrapper, jre, xlibs, zlib, nss, nspr, fontconfig, gnome2, cairo, expat, alsaLib, cups, dbus, atk, gtk2-x11, gdk_pixbuf, glib, curl, flite, freetype, libpulseaudio, systemd, libXxf86vm
```

Comment:
Please list these one per line
```
{ stdenv,
, fetchurl
, dpkg
...
```
Pretty much the style we've converged on.

Reply:
Thanks, I had forgotten about that ^-^'